client/my-sites/earn/memberships/index.tsx

```diff
@@ -618,7 +658,7 @@ function MembershipsSection( { query } ) {
 						</p>
 						<p>
 							{ preventWidows(
-								englishLocales.includes( getLocaleSlug() ) ||
+								englishLocales.includes( locale ) ||
```

Comment:
Replaced getLocaleSlug() because .includes() requires a string argument, and getLocaleSlug() returns string or null. Null is not a valid value here, and you also can't, say, pass an empty string if getLocaleSlug() returns null.

Reply:
You can use:

```js
import { useIsEnglishLocale } from '@automattic/i18n-utils';
const isEnglishLocale = useIsEnglishLocale();
```

Reply:
Oh, nice. Cleaner method.

That said, based on your other comments, looks like we can remove the conditional translations from this file, which I've done. Given that, we no longer need to check isEnglishLocale. So I've just removed this.